### ISSUE TITLE
Contact relation is not symmetric

### ISSUE THREAD

**Issue by LinqLover** [hpi-swt2/contacts-portal#125]
Let Alice add Bob to her contact list, then Alice will see Bob in her contact list, but Bob will not see Alice in his one. This is not really what I would have expected as Bob after accepting the contact request by Alice.

**Comment by frcroth** [comment]
I think you mean "symmetric"

**Comment by LinqLover** [comment]
Of course, sorry

**Comment by felixauringer** [comment]
If we think at the contact list in our phone, that would also be non-symmetric, but you're right, the contact requests may be confusing in this context. I think that should be clarified with a PO.

**Comment by j-hellenberg** [comment]
I agree, we also already thought about that. Since we are sending contact requests instead of just adding others to our contact book as we would do with phone contacts, anything else than a symmetric relation does not make too much sense to me

**Comment by LinqLover** [comment]
Btw these contact requests do not make great sense atm unless there is some "kind of private" data in your profile that is only visible to contacts ... Is something like this planned? :-)

**Comment by j-hellenberg** [comment]
I cannot currently say what's planned or not planned because one of our mistakes in the past was to rely on and create issues following too much only our own understanding of the vision. Our thinking was too much issue- instead of user-process-centered for the customer to really participate. I think this might be because in other courses like SWT I mainly the students had up to come with the features, while here the/one main input is/should be originating from the customer.

The customer will ultimately decide which end-to-end process(es) he wants to see implemented in the remainder of the time. I really wish we had been able to come up with that learning earlier because this might mean that some of the things we implemented so far will not be immediately used and might never be in the very limited time of our project.
